CMakeLists.txt

```diff
@@ -106,10 +106,13 @@ elseif(BASEARCH_X86_FOUND)
     option(WITH_SSE2 "Build with SSE2" ON)
     option(WITH_SSE4 "Build with SSE4" ON)
     option(WITH_PCLMULQDQ "Build with PCLMULQDQ" ON)
+elseif(BASEARCH_PPC_FOUND)
```

Comment:
Should be alphabetically. `BASEARCH_PPC_FOUND` before `BASEARCH_X86_FOUND`.

Reply:
Ok, but I'll put it before `BASEARCH_S390_FOUND` then.